src/Service/ServiceUtils.cpp

```diff
@@ -48,6 +48,11 @@ using ::orbit_object_utils::ObjectFile;
 
 static const char* kLinuxTracingEventsDirectory = "/sys/kernel/debug/tracing/events/";
 
+const std::vector<fs::path>& kHardcodedSearchDirectories{
+    "/home/cloudcast/",  "/home/cloudcast/debug_symbols/",  "/home/cloudcast/symbols",
```

Comment:
Do we need those here? It feels a bit strange to hardcode them. How was that done before?

Reply:
They were always hardcoded, I moved the location (from a default argument to this constant) and amended them a bit. The reason for amending them can be found here http://b/193199642

Reply:
Thanks, I missed that.